lasagne/layers/conv.py

```diff
@@ -933,6 +935,207 @@ def convolve(self, input, **kwargs):
 Deconv2DLayer = TransposedConv2DLayer
 
 
+class TransposedConv3DLayer(BaseConvLayer):
```

Comment:
Ah, we need a `# pragma: no cover` after the class definition as well, because it cannot be tested with the Theano version used on Travis.